creusot-contracts/src/std/ptr.rs

```diff
@@ -87,9 +96,9 @@ pub trait PointerExt<T: ?Sized>: Sized {
 }
 
 impl<T: ?Sized> PointerExt<T> for *const T {
-    #[logic(opaque)]
+    #[logic(open)]
```

Comment:
```suggestion
    #[logic(opaque)]
    #[cfg_attr(target_pointer_width = "16", builtin("creusot.prelude.Opaque.addr_logic_u16"))]
    #[cfg_attr(target_pointer_width = "32", builtin("creusot.prelude.Opaque.addr_logic_u32"))]
    #[cfg_attr(target_pointer_width = "64", builtin("creusot.prelude.Opaque.addr_logic_u64"))]
```

(You no longer need the `addr_logic` proxy.)